libs/headless-lfp/src/language-font-picker.ts

```diff
@@ -8,21 +8,23 @@ import { convertToFontLFP } from "./utils";
 export interface UseLanguageFontPicker {
   fetchFonts: (language: string) => Promise<void>;
   fonts: FontLFP[];
+  toggleSelectFont: (font: string) => boolean | undefined;
```

Comment:
nit: this function might read better if it's named `toggleSelectedFont`.

Reply:
`toggleFontSelected` makes more sense to me.